README.md

```
@@ -32,15 +32,28 @@ docker compose --profile broker --profile runner --profile nginx --profile front
-| broker | 8080 | 制御プレーン (DynamoDB) |
-| runner | 3000 | コマンド実行サーバー |
+| front | 5173 | Vite dev server (React + TypeScript) |
+| nginx | 8080 | リバースプロキシ |
+| broker | 8080 | 制御プレーン |
+| runner | 3000 | コマンド実行サーバ |
 | presenter | — | スライド同期・アンケート (AWS Lambda + API Gateway WebSocket) |
+| dynamodb-local | 8000 | ローカル開発用 DynamoDB |
+
+## アーキテクチャ
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)